### PR TITLE
Switch of PR check deployment from user/pass to token autentication.

### PR DESCRIPTION
**Pull request by rhopp**
### What does this PR do?
We used htpasswd authorization to login to dev cluster in PR check job to deploy rh-che there.
This PR switches to use service account and login using token, as htpasswd will be disabled in near future.

I've also added `edit` role to rh-che dev&qa users to be able to access logs etc.
@amisevsk @ibuziuk @davidfestal @garagatyi @ScrewTSW @Katka92 Please check your dev cluster username on line 135 in `cico_rhche_prcheck.sh` in this PR.

### What issues does this PR fix or reference?
https://github.com/redhat-developer/che-functional-tests/issues/323

### How have you tested this PR?
This PR is hard to test without submitting actual PR ;-)

### CODE REVIEW

File: .ci/cico_rhche_prcheck.sh [hunk 1]
```diff
@@ -136,6 +132,8 @@ else
 fi
 set -x
 
+oc policy add-role-to-user edit kkanova rhopp tdancs garagatyi ibuziuk amisevsk dfestal -n prcheck-${RH_PULL_REQUEST_ID}
```
Comment (davidfestal) [inline .ci/cico_rhche_prcheck.sh]: My GitHub user name (which is used in dev-cluster now afaict) is `davidfestal` (instead of `dfestal`)

Comment (rhopp) [reply .ci/cico_rhche_prcheck.sh]: Copy that!

Comment (rhopp) [reply .ci/cico_rhche_prcheck.sh]: @davidfestal Fixed.

**Comment by rhopp** [comment]
Confirmed working by David, Ilya, Angel and me. Merging.

**Pull request merged**